libraries.lisp

```diff
@@ -11,7 +11,8 @@
 (eval-when (:compile-toplevel :load-toplevel :execute)
   (cffi:define-foreign-library grpc-client-wrapper
 	;; Load lib on mac
-	(:darwin (:default "libgrpc"))
+	(:darwin #.(namestring
+				(asdf:system-relative-pathname "grpc" "grpc.so")))
```

Comment:
Why the newline?

Reply:
Do you mean that line 15 and line 14 can merge together?

Reply:
yes

Reply:
No specific reason. Good catch. I just formatted it, how is it now?

Reply:
lgtm